src/webhooks/overview.mdx

```diff
@@ -0,0 +1,74 @@
+---
+name: Overview
+menu: WEBHOOKS
+route: /docs/webhooks/overview
+---
+
+import { Title, Code, TextBlock } from "components"
+import webhook_settings from 'assets/webhook_settings.png'
+
+<Title>Overview</Title>
+Webhooks are sent from anomalies. Anomalies are described <a href="https://updates.smartlook.com/worried-about-anomalies-in-your-events-190172" target="_blank">HERE</a>
+
+# Cookbook
+
+<h3> 1. Set webhook URL on anomalies </h3>
+
+Optionaly set secret
+
+<img visibleOn="android,ios" src={webhook_settings} />
+
+<h3> 2. Set API on the server side </h3>
+
+Webhooks are sent as POST HTTP request. In the section [parameters](#parameters) is more information about webhooks message.
+
+
+<h3> 3. Verify the message </h3>
+
+You can verify the message about HASH. More info is below.
+
+# Security
```

Comment:
Tady bych to klidne vynechal a nechal tady jen ten popis "When your secret token...".